doc/en/components/layouts/dock-manager-customization.md

```diff
@@ -0,0 +1,90 @@
+---
+title: {Platform} Dock Manager | Customization | Infragistics
+_description: Use Infragistics' {Platform} dock manager component to manage the layout through panes, with the ability to customize it. Check out {ProductName} dock manager tutorials!
+_keywords: dock manager, layout, customization, {ProductName}
+mentionedTypes: ['DockManager']
+---
+
+### Customizing {Platform} Dock Manager
```

Comment:
This section's title needs to be the same size as the others in the topic: '## Customizing {Platform} Dock Manager'